File: angular_workspace/pfiva-web/src/app/data-model/survey/Option.ts
export class Option {

	private id: number;
	private value: string;

	public get Id(): number {
		return this.id;
	}
	public set Id(value: number) {
		this.id = value;
	}
	public get Value(): string {
		return this.value;
	}
	public set Value(value: string) {
		this.value = value;
	}
}

File: workspace/data-model/src/main/java/de/pfiva/data/model/survey/Option.java
package de.pfiva.data.model.survey;

import java.io.Serializable;

public class Option implements Serializable {

	private static final long serialVersionUID = 1L;
	
	private int id;
	private String value;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getValue() {
		return value;
	}
	public void setValue(String value) {
		this.value = value;
	}
	@Override
	public String toString() {
		return "Option [id=" + id + ", value=" + value + "]";
	}
}
